src/main/java/jdbchomework/dao/hibernate/DevelopersHibDao.java

```diff
@@ -2,11 +2,39 @@
 
 import jdbchomework.dao.model.DevelopersDao;
 import jdbchomework.entity.Developer;
+import jdbchomework.exceptions.DbConnectionException;
+import org.hibernate.HibernateException;
+import org.hibernate.Session;
 import org.hibernate.SessionFactory;
+import org.slf4j.LoggerFactory;
 
 
 public class DevelopersHibDao extends AbstractHibDao<Developer> implements DevelopersDao {
+
+    private org.slf4j.Logger log = LoggerFactory.getLogger(AbstractHibDao.class);
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png 'Severity: MINOR') Remove the "log" field and declare it as a local variable in the relevant methods. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AS1450)